lib/BIGSdb/Application.pm

```diff
@@ -350,21 +373,24 @@ sub read_config_file {
 }
 
 sub _read_db_config_file {
-	my ( $self, $config_dir ) = @_;
-	my $logger  = get_logger('BIGSdb.Application_Initiate');
+        my ( $self, $config_dir ) = @_;
+	my $logger = get_logger('BIGSdb.Application_Initiate');
 	my $db_file = "$config_dir/db.conf";
-	if ( !-e $db_file ) {
-		return;
+	if ( ! -e $db_file ){
+	    $logger->error("Couldn't find db.conf in $config_dir");
```

Comment:
db.conf is an optional file so you shouldn't get an error if it is not present. You could log this at a lower level, e.g. $logger->info and set your logging in /etc/bigsdb.conf to use this more verbose logging level, but it isn't an error.